File: front-end/src/app/service-models/app/v1/client/client-company-department-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class ClientCompanyDepartmentSM extends CoinManagementServiceModelBase<number> {
    departmentName!: string;
    departmenntLocation!: string;
    departmentDescription!: string;
    departmentCode!: string;
    departmentContact!: string;
    departmentManager!: string;
    clientCompanyDetailId!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/ClientCompanyDepartmentSM.cs
namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class ClientCompanyDepartmentSM : Base.SiffrumPayrollServiceModelBase<int>
    {
        public string DepartmentName { get; set; }
        public string DepartmenntLocation { get; set; }
        public string DepartmentDescription { get; set; }
        public string DepartmentCode { get; set; }
        public string DepartmentContact { get; set; }
        public string DepartmentManager { get; set; }
        public int ClientCompanyDetailId { get; set; }
    }
}
